zookeeper/1.7/Dockerfile

```diff
@@ -6,9 +6,9 @@ LABEL maintainer="Debezium Community"
 # Set the version, home directory, and SHA hash.
 # SHA 512 hash from https://www.apache.org/dist/zookeeper/zookeeper-$ZK_VERSION/zookeeper-$ZK_VERSION.tar.gz.sha512
 #
-ENV ZK_VERSION=3.5.8 \
+ENV ZK_VERSION=3.5.9 \
```

Comment:
Thx for that update; I actually should have done that as part of the AK update to 2.8.0.